src/gui/rss/rssparser.cpp

```diff
@@ -266,6 +272,11 @@ void RssParser::parseRssArticle(QXmlStreamReader& xml, const QString& feedUrl)
         else
           article["news_link"] = link;
       }
+      else if(xml.name() == "magnetURI") {
```

Comment:
Sorry for the late reply, but yeah I was trying to get the kat.cr RSS feed working.
So the reason I gave priority to magnet links is that they seems easier to 'verify' if they are a valid link. The torrent link has less guarantees, it's just a link so the serve can be unavailable, the contents of the file it points to might change. So it seems like the magnet was a safer bet.
But I am not super familiar to the cons and pros of a magnet vs a torrent file :)